synapse/handlers/profile.py

```diff
@@ -99,13 +100,13 @@ def _replicate_host_profile_batch(self, host, batchnum):
         logger.info("Replicating profile batch %d to %s", batchnum, host)
         batch_rows = yield self.store.get_profile_batch(batchnum)
         batch = {
-            UserID(r["user_id"], self.hs.hostname).to_string(): {
+            UserID(r["user_id"], self.hs.hostname).to_string(): ({
                 "display_name": r["displayname"],
                 "avatar_url": r["avatar_url"],
-            } for r in batch_rows
+            } if r["active"] else None) for r in batch_rows
         }
 
-        url = "https://%s/_matrix/identity/api/v1/replicate_profiles" % (host,)
+        url = "http://%s/_matrix/identity/api/v1/replicate_profiles" % (host,)
```

Comment:
hm, why?

Reply:
oops, because I didn't mean to commit that